### PR TITLE
Add details and minor edits regarding windows compatibility

### PR DESCRIPTION
**Pull request by CameronTEllis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To build code that satisfies these goals is no easy feat. The current repository
...
 
-This has been tested on PTB 3.0.X on Macs and Linux and MATLAB versions newer than 2014a. Mileage may vary by system, send feedback if you have compatibility issues. 
+This has been tested on PTB 3.0.X on Macs and Linux and MATLAB versions newer than 2014a. In accordance with the advice from PTB it is recommended that you use a Linux system for running this code to ensure timing. In addition to the basic PTB install you will also need to install gstreamer in order to play movies. Gstreamer can be difficult to setup, in my experience version 1.14.4 works best with PTB. With all of this, mileage may vary by system, send feedback if you have compatibility issues.
```
Comment (tsy6) [inline README.md:17]: any particular version of gstreamer?

File: Scripts/GenerateTrials_PlayVideo.m [hunk 2]
```diff
@@ -12,7 +12,17 @@
...
-Parameters.BaseExt=Parameters.BaseExt(1:max(find(Parameters.BaseExt=='/'))); %Remove the current folder
+if ispc == 1
+    Parameters.BaseExt=Parameters.BaseExt(1:max(find(Parameters.BaseExt=='\'))); %Remove the current folder
+else
+    Parameters.BaseExt=Parameters.BaseExt(1:max(find(Parameters.BaseExt=='/'))); %Remove the current folder
+end
+
+% Test whether you got a reasonable result
+if isempty(Parameters.BaseExt)
+    warning('Parameters.BaseExt is length zero, probably because your computer uses a different file system path that what is assumed');
```
Comment (tsy6) [inline Scripts/GenerateTrials_PlayVideo.m:23]: **than what is assumed

File: Scripts/GenerateTrials_PlayVideo.m [hunk 3]
```diff
@@ -12,7 +12,17 @@
 %Set up the parameters of the experiment
 
 Parameters.BaseExt=cd; %What is the current folder directory name
-Parameters.BaseExt=Parameters.BaseExt(1:max(find(Parameters.BaseExt=='/'))); %Remove the current folder
+if ispc == 1
```
Comment (tsy6) [inline Scripts/GenerateTrials_PlayVideo.m:15]: where is ispc defined?

**Review by tristansyates — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (CameronTEllis) [reply README.md:17]: I state that version 1.14.4 is best, is that not clear?

File: Scripts/GenerateTrials_PlayVideo.m (hunk 2 above)
Comment (CameronTEllis) [reply Scripts/GenerateTrials_PlayVideo.m:23]: Good catch

File: Scripts/GenerateTrials_PlayVideo.m (hunk 3 above)
Comment (CameronTEllis) [reply Scripts/GenerateTrials_PlayVideo.m:15]: `ispc` is a matlab call introduced in 2006a: https://www.mathworks.com/help/matlab/ref/ispc.html

File: README.md (hunk 1 above)
Comment (tsy6) [reply README.md:17]: oh yes it is i just missed it

File: Scripts/GenerateTrials_PlayVideo.m (hunk 3 above)
Comment (tsy6) [reply Scripts/GenerateTrials_PlayVideo.m:15]: Ahh nice!

**Review by tristansyates — APPROVED** [review]

**Pull request merged**